docker/README.md

```diff
@@ -17,4 +17,7 @@ OpenOnDemand is via the `ood` user with password `ood`
 
 ## Development
 
-    docker build -t ohiosupercomputer/ood:1.5.0 -t ohiosupercomputer/ood:latest .
\ No newline at end of file
+    cp -f ../common/ood-setup.sh ./ood-setup.sh
+    docker build -t ohiosupercomputer/ood:1.6.2 -t ohiosupercomputer/ood:latest .
+    docker push ohiosupercomputer/ood:1.6.2
```

Comment:
shouldn't the docker images have the same version as the underlying on-demand? i.e., 1.6.17.  Or maybe 3 tags as `latest`, `1.6` and `1.6.17`?

Reply:
That's not historically what I've done because I don't intend people to deploy OnDemand using these containers, just for playing around with. We'd need to rework a lot of container setup to make it something that a site could use to actually run OnDemand in production.

Reply:
I'm rebuilding the containers now with tags like you suggested but we need to be careful not to advertise this container as a way to run ondemand in production.

Reply:
OK yea, not sure how the two are related, but I get the bit about not wanting this in production.